### DISCUSSION TITLE
Azure Purgatory

### DISCUSSION THREAD

**Discussion by Blake-Izayoi** [nzp-team/nzportable#898] [Map Releases]
My first original map release. Uses the ikblue.wad as well as an expanded pack of said wad as the main theming. Has the World at War perks, Mule Kick, and Pack-a-Punch. Mule Kick & Pack-a-Punch are locked behind an easter egg. Tight corridors and hallways make it so you have to have a plan before you start running around on high rounds. The map has been tested on both the 3DS as well as PC, although it looks a bit ugly on the 3DS. Let me know your thoughts, thank you.
Credits: NZ:P Team, Sock
Version 1: Release!
Version 1.1: Removed troublesome water that didn't render properly on the PlayStation Vita.
### [DOWNLOAD](https://archive.org/download/azure-purgatory-nzp/AzurePurgatory.zip)

![image](https://github.com/user-attachments/assets/6f487078-d54b-4c5f-b5c9-0979a4dcb069)
![image](https://github.com/user-attachments/assets/7db7abe4-744d-476a-b163-35359c559cfd)
![image](https://github.com/user-attachments/assets/2bdba5b8-fc25-42ce-821d-b3dd3013f1f4)
![image](https://github.com/user-attachments/assets/84b75329-6115-4e57-812f-dd6c0c934022)

**Comment by MotoLegacy** [comment]
this looks very cool!

**Reply by Blake-Izayoi** [reply]
Thanks a ton, that means a lot!

**Comment by ToonLink0001** [comment]
this have an ee song?

**Reply by Blake-Izayoi** [reply]
No, sorry.

**Comment by bignoobus** [comment]
This map is actually sick, one of the better maps for sure, just a single pet peeve tho, the waypointing is a bit uhm, to put it lightly its dodgy, but other than that its beautiful
I like how pack a punch is locked behind an easter egg, a tough one at that, im still trying to find the last teddy xD its like a reward, very nice!

**Reply by Blake-Izayoi** [reply]
Thank you for the kind words! What do you think I could change about the waypointing to make it better?

**Reply by bignoobus** [reply]
I have... No clue actually 😅, they usually walk into walls alot tho, which is a shame since the map is extremely good, I'll experiment with the waypoints tmr to see if I can spot anything

**Comment by Jeyphr** [comment]
Zaaaamn this looks good! It reminds me of those artsy quake levels.

**Reply by Blake-Izayoi** [reply]
Thank you a ton! This level did take some inspiration from Quake. I think Quake's theming and architecture is really amazing and I'd like to make more maps based around it.

**Comment by DerpedCrusader** [comment]
Absolutely love the vibes, and detail work 

I've noticed there's a lot of faulty spawning, this can happen due to spawns being too close to walls, but I'm not certain without looking at the source files

I can tell it's a spawning issue, and not a waypoint issue, because when I walk next to them, they don't path to me, meaning they're not marked as inside the play space

**Comment by Phynix0027** [comment]
I had a problem, I found all the 3 teddys, but the door did not opened. I am on the 3ds version

**Comment by MotoLegacy** [comment]
Tagging as Meets Install Standard!

**Comment by Jeyphr** [comment]
Check out my review right here! :https://youtu.be/z9O5_vMFq6w

**Comment by nonMat** [comment]
This was really good! 
The only thing is that sometimes the zombies would spawn outside of the barricades.
Still great tho!
